zircon.core/src/commonMain/kotlin/org/hexworks/zircon/api/fragment/Table.kt

```diff
@@ -0,0 +1,31 @@
+package org.hexworks.zircon.api.fragment
+
+import org.hexworks.cobalt.databinding.api.value.ObservableValue
+import org.hexworks.zircon.api.component.Fragment
+import org.hexworks.zircon.api.data.Size
+
+/**
+ * A table fragment displays data in rows. Each row contains several cells. How a cell is displayed depends
+ * on the definition of it's column.
+ *
+ * A table contains data of type [M] (the model). Every model object represents one row in the table. **A table
+ * may not be empty**.
+ *
+ * @param M the type of the underlying model representing this table's data.
+ */
+interface Table<M: Any>: Fragment {
```

Comment:
`Table<M: Any>`

I see the missing space here as a recurring issue in this change. I'd double-check you're using the official [code guidelines](https://kotlinlang.org/docs/coding-conventions.html#configure-style-in-ide) in IntelliJ, because the [recommendation](https://kotlinlang.org/docs/coding-conventions.html#colon) is to format this as `Table<M : Any>`. Minor, for sure, but it's good to be consistent.

Reply:
Yup I totally agree. I just got used to refrain from formatting whole files. That leads to some lines sneaking through with ugly parts ;)